base/terminfo_data.jl

```diff
@@ -41,9 +41,9 @@ version_info = IOBuffer()
 standard_caps = IOBuffer()
 user_caps = IOBuffer()
 
-Downloads.download("https://raw.githubusercontent.com/mirror/ncurses/master/VERSION", version_info)
-Downloads.download("https://raw.githubusercontent.com/mirror/ncurses/master/include/Caps", standard_caps)
-Downloads.download("https://raw.githubusercontent.com/mirror/ncurses/master/include/Caps-ncurses", user_caps)
+Downloads.download("https://raw.githubusercontent.com/ThomasDickey/ncurses-snapshots/refs/heads/master/VERSION", version_info)
+Downloads.download("https://raw.githubusercontent.com/ThomasDickey/ncurses-snapshots/refs/heads/master/include/Caps", standard_caps)
+Downloads.download("https://raw.githubusercontent.com/ThomasDickey/ncurses-snapshots/refs/heads/master/include/Caps-ncurses", user_caps)
```

Comment:
So https://github.com/mirror/ncurses was last updated two years ago, by Thomas Dickey who is the ncurses maintainer. Based on this and some additional prodding, new repo seems to be a good replacement